tools/aot/aot_compile.bzl

```diff
@@ -242,7 +242,7 @@ def aot_compile(
                 "//tools/aot:abi",
                 "@cnpy//:cnpy",
                 "@com_google_googletest//:gtest_main",
-                "@llvm-project//mlir:mlir_c_runner_utils_hdrs",
+                "@llvm-project//mlir:mlir_c_runner_utils",
```

Comment:
This is needed for `memrefCopy` symbol ([source](https://sourcegraph.com/github.com/llvm/llvm-project@762cbd82da4debf8b026a4eb4ade66720acf3182/-/blob/mlir/lib/ExecutionEngine/CRunnerUtils.cpp?L62:17-62:27)).